sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -805,12 +906,14 @@ static int sof_card_dai_links_create(struct device *dev,
 #endif
 	struct snd_soc_dai_link_component *ssp_components;
 	struct snd_soc_acpi_mach_params *mach_params;
-	int ssp_num, sdw_num, hdmi_num = 0, dmic_num;
+	int ssp_num, sdw_num = 0, hdmi_num = 0, dmic_num;
 	const struct snd_soc_acpi_link_adr *adr_link;
 	struct snd_soc_dai_link_component *cpus;
+	bool group_generated[MAX_SDW_LINKS];
 	struct snd_soc_dai_link *links;
 	int num_links, ssp_mask;
 	char *name, *cpu_name;
+	int cpu_dai_num = 0;
```

Comment:
is the initialisation needed? Can `get_sdw_dai_info()` return 0 and not set `cpu_dai_num`?